content/advanced/330_servicemesh_using_appmesh/port_to_app_mesh/create_meshed_app.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Create the Meshed Application"
+date: 2020-01-27T08:30:11-07:00
+weight: 50
+---
+
+#### Create Mesh Object
+
+Configure namespace with App Mesh Labels and deploy Mesh Object
+
+```bash
+kubectl apply -f deployment/mesh.yaml  
+```   
+{{< output >}}
+namespace/prodcatalog-ns configured 
+mesh.appmesh.k8s.aws/prodcatalog-mesh created
+{{< /output >}}
+
+Confirm the Mesh object and Namespace are created
+
+```bash
+kubectl describe namespace prodcatalog-ns
+```    
+{{< output >}}
+Name:         prodcatalog-ns
+Labels:       appmesh.k8s.aws/sidecarInjectorWebhook=enabled
+            gateway=ingress-gw
```

Comment:
maybe add a note below describing the gateway label, as it's later used for the Vgateway?

Reply:
updated